sdks/java/core/src/main/java/org/apache/beam/sdk/options/SdkHarnessOptions.java

```diff
@@ -386,4 +387,20 @@ static List<Logger> getConfiguredLoggerFromOptions(SdkHarnessOptions loggingOpti
     }
     return configuredLoggers;
   }
+
+  @Description(
```

Comment:
I suggest `@Hidden` so that this is not shown to users when they do `--help`

Reply:
Done!